### DISCUSSION TITLE
package search feature

### DISCUSSION THREAD

**Discussion by shivasai09** [codesandbox/codesandbox-client#4364] [General]
how did codesandbox added all the packages which are available via npm  in the algolia index.

i believing that we need to add all the packages which are available in npm in the algolia index. then we can search for the package very easily

by seeing the below code  i am assuming . we have all the data in that index.. 
![image](https://user-images.githubusercontent.com/33802398/84565461-f5d7b980-ad86-11ea-9ce2-5d3f47b901d8.png)

my only question is , where do  we get that data, to add to the index.. is there any external api which provides package information

**Comment by lbogdan** [comment]
Maybe this: https://github.com/algolia/npm-search ?
